aiida/orm/querybuilder.py

```diff
@@ -1710,32 +1710,6 @@ def _get_connecting_node(self, index, joining_keyword=None, joining_value=None,
                                                    self.tag_to_alias_map.keys()))
         return returnval
 
-    def _get_json_compatible(self, inp):
```

Comment:
Since we are now removing this function, could you please:

* Move the implementation from `get_json_compatible_queryhelp` into the `queryhelp` property
* Use `self.queryhelp` in `get_json_compatible_queryhelp` and add a deprecation warning (will be removed in AiiDA 2.0)?

Reply:
Hereby done. Tried to assimilate the doc string as well a bit.